cerberus/cerberus.go

```diff
@@ -60,6 +61,31 @@ func NewClient(authMethod auth.Auth, otpFile *os.File) (*Client, error) {
 		CerberusURL:    authMethod.GetURL(),
 		vaultClient:    vclient,
 		httpClient:     &http.Client{},
+		defaultHeaders: http.Header{},
+	}, nil
+}
+
+func NewClientWithHeaders(authMethod auth.Auth, otpFile *os.File, defaultHeaders http.Header) (*Client, error) {
```

Comment:
- [ ] Add unit test to test that you can use your new feature and that the old logic still works.